app/models/spree/order.rb

```diff
@@ -720,7 +720,7 @@ def remove_variant(variant)
     end
 
     def cap_quantity_at_stock!
-      line_items.includes(variant: :stock_items).all.find_each(&:cap_quantity_at_stock!)
+      line_items.includes(variant: :stock_items).all.each(&:cap_quantity_at_stock!)
```

Comment:
have you tried removing `all` and sticking to `find_each`?

Reply:
I will try now and watch the build :+1:

Reply:
done :+1:  it works.